content/docs/glossary/multisig.mdx

```diff
@@ -69,10 +73,18 @@ This type of signer is especially useful in escrow accounts. You can pre-authori
 
 ### Hash(x)
 
-Adding a signature of type hash(x) allows anyone who knows `x` to sign the transaction. This type of signer is especially useful in [atomic cross-chain swaps](https://en.bitcoin.it/wiki/Atomic_cross-chain_trading) which are needed for inter-blockchain protocols like [lightning networks](https://lightning.network).
+Adding a signer of type hash(x) allows anyone who knows `x` to sign the transaction. This type of signer is especially useful in [atomic cross-chain swaps](https://en.bitcoin.it/wiki/Atomic_cross-chain_trading) which are needed for inter-blockchain protocols like [lightning networks](https://lightning.network).
 
 First, create a random 256 bit value, which we call `x`. The SHA256 hash of that value can be added as a signer of type hash(x). Then in order to authorize a transaction, `x` is added as one of the signatures of the transaction. Keep in mind that `x` will be known to the world as soon as a transaction is submitted to the network with `x` as a signature. This means anyone will be able to sign for that account with the hash(x) signer at that point. Often you want there to be additional signers so someone must have a particular secret key and know `x` in order to reach the weight threshold required to authorize transactions on the account.
 
+### Signed Payloads
+
+Adding a signed payload as a signer means that transactions with a signature on that payload by that signer are valid.
```

Comment:
Suggest some changes for subtle clarity:
```suggestion
Adding a signed payload as a signer means that transactions with a signature, of a specified payload by a specified key, are valid.
```

Reply:
This is more confusing to me. "a specified" feels generic: specified where? whereas "that payload" is a back-reference to the prior mention of payload. Maybe "**the** specified"? And "specified key" is technically wrong since it's the private key that needs to sign, but it's not the one that's specified.. :laughing: this is hard to explain precisely

Reply:
How about:
```suggestion
Adding a signed payload as a signer means that transactions are only valid if they are accompanied by a signature of the signer's payload. The signature must be produced using the private key corresponding to the public key specified in the signer.
```